packages/terra-clinical-label-value-view/src/LabelValueView.jsx

```diff
@@ -45,12 +45,26 @@ const LabelValueView = ({
     textValueSection = <div className={cx('value')}>{textValue}</div>;
   }
 
+  if (customProps.childOfDescriptionList === 'true') {
+    return (
+      <React.Fragment>
```

Comment:
this wrapper is missing  cutomProps . I am not sure if we can do it on react fragment

Reply:
We return the fragment only when the LabelValueView is part of a definition list. In that case, the consumer will be responsible for adding properties/classes to the definition list. I already tried to add the customProps on the fragment, but that is not getting applied on the definition list (the parent of the LabelValueView). I discussed this with the person working on the ClinicalDetailView (a component that is consuming the LabelValueView and is adding it as part of a definition list) and said that they are going to set the properties on their definition list. Do you think we should rethink this scenario?

Reply:
do you mean in ClinicalDetailView in LabelValueView is rendered within <dl > element ??

Reply:
I have some questions about the introduction of the new prop so we may need to revisit this discussion. 

CC: @chrismichalewicz , @eawww

Reply:
Note: the questions and concerns got resolved after an offline conversation.